docs/content/connectors/mysql-cdc.md

```diff
@@ -52,29 +53,24 @@ Notes
+keeps its state(the binlog position and corresponding table schema), thus binlog phase can support checkpoint in binlog position level.
 
-### Set a differnet SERVER ID for each job
-Every MySQL database client for reading binlog should have an unique id, called server id. MySQL server will use this id to maintain network connection and the binlog position. Therefore, if different jobs share a same server id, it may result to reading from wrong binlog position. 
-Thus, it is recommended to set different server id for each job via the [SQL Hints](https://ci.apache.org/projects/flink/flink-docs-release-1.11/dev/table/sql/hints.html), e.g. `SELECT * FROM source_table /*+ OPTIONS('server-id'='123456') */ ;`. 
-
-Tip: By default the server id is randomed when the TaskManager is started up. If TaskManager is failed down, it may have a different server id when starting up again. But this shouldn't happen frequently (job exception doesn't restart TaskManager), and shouldn't affect MySQL server too much.
+Flink will periodically perform checkpoints for the source, in case of failover, the job will restart and restore from the checkpointed state, it guarantees the exactly once semantic. 
 
-### Can't perform checkpoint during scaning database tables
-During scanning tables, since there is no recoverable position, we can't perform checkpoints. In order to not perform checkpoints, MySQL CDC source will keep the checkpoint waiting to timeout. The timemout checkpoint will be recognized as failed checkpoint, by default, this will trigger a failover for the Flink job. So if the database table is large, it is recommended to add following Flink configurations to avoid failover because of the timeout checkpoints:
+### Set a different SERVER ID for each reader
+Every MySQL database client for reading binlog should have an unique id, called server id. MySQL server will use this id to maintain network connection and the binlog position. Therefore, if different jobs share a same server id, it may result to reading from wrong binlog position. 
+Thus, it is recommended to set different server id for each reader via the [SQL Hints](https://ci.apache.org/projects/flink/flink-docs-release-1.11/dev/table/sql/hints.html), 
+e.g. Assume the source parallelism is 4 which means we have 4 readers, then we can use `SELECT * FROM source_table /*+ OPTIONS('server-id'='5401,5404') */ ;`. 
 
-```
-execution.checkpointing.interval: 10min
-execution.checkpointing.tolerable-failed-checkpoints: 100
-restart-strategy: fixed-delay
-restart-strategy.fixed-delay.attempts: 2147483647
-```
+Tip: By default the server id is random when the TaskManager is started up. If TaskManager is failed down, it may have a different server id when starting up again. But this shouldn't happen frequently (job exception doesn't restart TaskManager), and shouldn't affect MySQL server too much.
```

Comment:
`By default the server id is deterministed randomly when job is started up`? I think the server id is not deterministed by TM.